kie-dmn-validation/src/main/resources/rules.drl

```diff
@@ -201,7 +201,8 @@ end
 
 rule NAME_INVALID
 when
-    $ne : NamedElement( FEELParser.isVariableNameValid( name ) == false )
+    $ne : NamedElement( parent != null, !(parent instanceof Decision),                       // << functional optimization to avoid double reporting as Decision name == variable name 
+                        FEELParser.isVariableNameValid( name ) == false )
```

Comment:
Unfortunately we also need to check for parents of types BusinessKnowledgeModel and InputData. There might be more, we need to double check.